exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Description
+
+Today, most people in the world use Arabic numbers.
+But if you travelled back two thousand years, you'd find that most Europeans were using Roman numerals instead.
+
+To write a Roman numeral we use the following Latin letters, each of which has a value:
+
+| M    | D   | C   | L  | X  | V | I |
+|------|-----|-----|----|----|---|---|
+| 1000 | 500 | 100 | 50 | 10 | 5 | 1 |
+
+A Roman numeral is a sequence of these letters, and its value is the sum of the letters' values.
+For example, `XVIII` has the value 18 (`10 + 5 + 1 + 1 + 1 = 18`).
+
+There's one rule that makes things tricker though, and that's that **the same letter cannot be used more than three times in succession**.
+That means that we can't express numbers such as 4 using the seamingly natural method of `IIII`.
+Instead, for those numbers, we use a subtraction method between two letters.
+So we think of `4` not as `1 + 1 + 1 + 1` but instead as `5 - 1`.
+And slightly confusingly to our modern thinking, we write the smaller number first.
+This applies in the following cases: 4 (`IV`), 9 (`IX`), 40 (`XL`), 90 (`XC`), 400 (`CD`) or 900 (`CM`).
+
```

Comment:
These rules do not rule out numerals like VVV. However, failing tests probably make amply clear what the intent is, so I do not foresee problems here.